grapher/stackedCharts/StackedAreaChart.tsx

```diff
@@ -159,16 +188,7 @@ class Areas extends React.Component<AreasProps> {
         const { hoverIndex } = this
 
         return (
-            <g
-                ref={this.base}
-                className="Areas"
-                onMouseMove={this.onCursorMove}
-                onMouseLeave={this.onCursorLeave}
-                onTouchStart={this.onCursorMove}
-                onTouchMove={this.onCursorMove}
-                onTouchEnd={this.onCursorLeave}
-                onTouchCancel={this.onCursorLeave}
-            >
+            <g ref={this.base} className="Areas">
```

Comment:
With @ikesau working on the faceting tooltips we're probably switching to event listeners attached to the chart's parent `<g>`, because otherwise if we attach to the parent `<svg>` hover events get triggered for all facets.

@marcelgerber What are your thoughts on it? Is it reasonable to leave the event listeners on `<g>` here, and maybe add an invisible `<rect>` to capture events?

(This is kind of a separate issue, but just trying to minimise conflicts.)

Reply:
The change in question: [grapher/lineCharts/LineChart.tsx:262](https://github.com/owid/owid-grapher/pull/1425/files#diff-12d2b06e51b904803cc44178eedbf5717e9d65d019eda13102111b9bb5d1fe7fR262)

Reply:
The faceting tooltips PR is not merged and this needs a rebase, and potentially we need to go back to adding event listeners through React? We just might need to inject an invisible `<rect>` to make them fire outside the plot?